### PR TITLE
05-02 emode update proposal

### PR DESCRIPTION
**Pull request by reem**
Implements https://snapshot.org/#/aave.eth/proposal/0x84deca82139320b2570f04211b249e37b8a7602b4a0ed70e6fa772c9f6e94550

Uses modified generator from `manual-emode-category-update` that generates the emode category update calls to the pool configurator directly in a `_postExecute` hook instead of using the config engine.

IPFS preview: https://app.aave.com/governance/ipfs-preview/?ipfsHash=0x5610976644e27ab476a22ab81a1f6b120eb992d5624a96a39bd76bf2737a1c3f

RR-120

### CODE REVIEW

**Comment by reem** [comment]
The tests are currently failing with:

```[FAIL. Reason: PROPOSAL_EXECUTION_FAILED] testPolygon20230502UpdatePayload() (gas: 10483939)```

And this is the end of the stack trace with -vvv
```
    │   │   │   │   │   ├─ emit EModeAssetCategoryChanged(param0: 0xa3Fa99A148fA48D14Ed51d610c367C61876997F1, param1: 1, param2: 0)
    │   │   │   │   │   └─ ← ()
    │   │   │   │   └─ ← ()
    │   │   │   ├─ [1255] 0x794a61358D6845594F94dc1DB02A252b5b4814aD::getConfiguration(0x4e3Decbb3645551B8A19f0eA1678079FCB33fB4c) [staticcall]
    │   │   │   │   ├─ [703] 0xDF9e4ABdbd94107932265319479643D3B05809dc::getConfiguration(0x4e3Decbb3645551B8A19f0eA1678079FCB33fB4c) [delegatecall]
    │   │   │   │   │   └─ ← 0x000000000000000000000103e800000000000000000007d00712000000000000
    │   │   │   │   └─ ← 0x000000000000000000000103e800000000000000000007d00712000000000000
    │   │   │   └─ ← 0x4e487b710000000000000000000000000000000000000000000000000000000000000011
    │   │   └─ ← "Arithmetic over/underflow"
    │   └─ ← "PROPOSAL_EXECUTION_FAILED"
    └─ ← "PROPOSAL_EXECUTION_FAILED"
```

**Review by WatFu — COMMENTED** [review]
Looks good from PPM/DS perspective.

**Comment by jmomx** [comment]
- https://polygonscan.com/address/0x32f3A6134590fc2d9440663d35a2F0a6265F04c4

1. emode cat -> 0 for mimatic, ageur, usdt, jeur, eurs (💡 Category 0 is reserved as the default non-eMode category. All the assets listed on Aave V3, by default have category 0 (which indicates the standard operational mode). https://docs.aave.com/developers/whats-new/efficiency-mode-emode)
2. for emode 1, ltv 95, lt 93 
✅ 

- https://snowtrace.io/address/0x0568a3aeb8e78262deff75ee68fac20ae35ffa91

1. emode 0 -> for mai and usdt
2. for emode 1, ltv 95, lt 93 
✅ 

- https://optimistic.etherscan.io/address/0x78Fe5d0427E669ba9F964C3495fF381a805a0487#code
1. emode 0 for usdt, susd
2. for emode 1, ltv 95, lt 93 
✅ 
- https://arbiscan.io/address/0x33DeAc0861FD6a9235b86172AA939E79085c6f52
- 1. emode 0 for usdt, eurs
2. for emode 1, ltv 95, lt 93 
✅

**Review by jmomx — APPROVED** [review]
looks good - see comments but verified https://app.aave.com/governance/ipfs-preview/?ipfsHash=0x5610976644e27ab476a22ab81a1f6b120eb992d5624a96a39bd76bf2737a1c3f against the contract on each network

File: diffs/preTestArbitrumUpdate20230502_postTestArbitrumUpdate20230502.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+## Reserve changes
+
+### Reserve altered
+
+#### EURS ([0xD22a58f79e9481D1a88e00c343885A588b34b68B](https://https://arbiscan.io/address/0xD22a58f79e9481D1a88e00c343885A588b34b68B))
```
Comment (reichiang) [inline diffs/preTestArbitrumUpdate20230502_postTestArbitrumUpdate20230502.md:5]: nit: remove duplicated "https://"

File: diffs/preTestArbitrumUpdate20230502_postTestArbitrumUpdate20230502.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+| description | value before | value after |
+| --- | --- | --- |
+| eModeCategory | 1 | 0 |
+
+
+#### USDT ([0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9](https://https://arbiscan.io/address/0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9))
```
Comment (reichiang) [inline diffs/preTestArbitrumUpdate20230502_postTestArbitrumUpdate20230502.md:12]: nit: remove duplicated "https://"

**Review by reichiang — APPROVED** [review]
lgtm

**Comment by reem** [comment]
Closing this PR to remove all stale reviews, will create a new PR with the updated proposal details

**Pull request closed without merging**